### ISSUE TITLE
NHM3 3.0.5.4 with 460.79 nvidia driver fails

### ISSUE THREAD

**Issue by EvgeniyKorepov** [nicehash/NiceHashMiner#2234]
```
[2020-12-09 21:11:56] [INFO] [NICEHASH] Starting up NiceHash Miner v3.0.5.4
[2020-12-09 21:11:56] [INFO] [NICEHASH] TOS differs! agreed: 0 != Current 4
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: en, name: English
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: ru, name: Русский (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: es, name: Español (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: pt, name: Português (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: bg, name: Български (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: it, name: Italiano (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: pl, name: Polski (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: zh_cn, name: 简体中文 (Unofficial)
[2020-12-09 21:11:57] [INFO] [Translations] Found language: code: ro, name: Română (Unofficial)
[2020-12-09 21:12:02] [INFO] [SystemSpecs] WMI service seems to be running, ManagementObjectSearcher returned success.
[2020-12-09 21:12:02] [INFO] [NHM] Get OS version START
[2020-12-09 21:12:02] [INFO] [NHM] Get OS version END
[2020-12-09 21:12:08] [INFO] [SystemSpecs] FreePhysicalMemory = 12230208, 11943 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] FreeSpaceInPagingFiles = 8818452, 8611 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] FreeVirtualMemory = 18451320, 18018 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] LargeSystemCache = 0, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] MaxNumberOfProcesses = 4294967295, 4194303 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] MaxProcessMemorySize = 137438953344, 134217727 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] NumberOfLicensedUsers = 0, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] NumberOfProcesses = 199, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] NumberOfUsers = 2, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] OperatingSystemSKU = 101, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] SizeStoredInPagingFiles = 8818452, 8611 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] SuiteMask = 784, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] TotalSwapSpaceSize = 0, 0 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] TotalVirtualMemorySize = 25544240, 24945 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] TotalVisibleMemorySize = 16725788, 16333 MB
[2020-12-09 21:12:08] [INFO] [SystemSpecs] PageFileSize = 8818452, 8611 MB
[2020-12-09 21:12:08] [INFO] [DeviceDetection] DetectCPU START
[2020-12-09 21:12:08] [INFO] [CPUDetector] TryQueryCUDADevicesAsync START cpu -n
[2020-12-09 21:12:08] [INFO] [CPUDetector] TryQueryCUDADevicesAsync END cpu -n
[2020-12-09 21:12:08] [INFO] [CPUDetector] {"AMD":true,"Intel":false,"IsZen":true,"L3KB_size":16384,"Name":"AMD Ryzen 5 1600 Six-Core Processor            ","PhysicalProcessorCount":1,"SupportsAES_SSE42":true,"SupportsAVX":true,"SupportsAVX2":true,"SupportsSSE2":true,"Vendor":"AuthenticAMD"}
[2020-12-09 21:12:09] [INFO] [DeviceDetection] Found CPU:
	UUID: CPU-a3d51b00-bfa9-5019-9937-1dae10dfe28e
	Name: AMD Ryzen 5 1600 Six-Core Processor
	PhysicalProcessorCount: 1
	ThreadsPerCPU: 12
	SupportsHyperThreading: True

[2020-12-09 21:12:09] [INFO] [DeviceDetection] DetectCPU END
[2020-12-09 21:12:09] [INFO] [VideoControllerDetector] QueryWin32_VideoControllerTask START
[2020-12-09 21:12:09] [INFO] [VideoControllerDetector] QueryWin32_VideoControllerTask END
[2020-12-09 21:12:09] [INFO] [DeviceBusDetector] QueryWin32_DeviceBusPCITask START
[2020-12-09 21:12:09] [INFO] [DeviceBusDetector] QueryWin32_DeviceBusPCITask END
[2020-12-09 21:12:09] [INFO] [DeviceDetection] NVIDIA driver version: 460.79
QueryVideoControllers: 
	Win32_VideoController detected:
		Name: NVIDIA GeForce RTX 3090
		Description: NVIDIA GeForce RTX 3090
		PNPDeviceID: PCI\VEN_10DE&DEV_2204&SUBSYS_40451458&REV_A1\4&1C3D25BB&0&0019
		DriverVersion: 27.21.14.6079
		Status: OK
		InfSection: Section087
		AdapterRAM: 4293918720
		PCI_BUS_ID: 7

[2020-12-09 21:12:09] [INFO] [CUDADetector] TryQueryCUDADevicesAsync START cuda -n
[2020-12-09 21:12:10] [INFO] [CUDADetector] TryQueryCUDADevicesAsync END cuda -n
[2020-12-09 21:12:10] [INFO] [DeviceDetection] TryQueryCUDADevicesAsync RAW: '{"CudaDevices":[{"DeviceGlobalMemory":25769803776,"DeviceID":0,"DeviceName":"GeForce RTX 3090","HasMonitorConnected":0,"SMX":82,"SM_major":8,"SM_minor":6,"UUID":"","VendorID":0,"VendorName":"","pciBusID":7,"pciDeviceId":0,"pciSubSystemId":0}],"DriverVersion":"","ErrorString":"","NvmlInitialized":1,"NvmlLoaded":1}'
[2020-12-09 21:12:10] [ERROR] [ApplicationStateManager.Init] Exception: Входная строка имела неверный формат.
```

**\NHM3\app_3.0.5.4\device_detection.exe cuda -n** 
Result :
```
{
	"CudaDevices": [{
		"DeviceGlobalMemory": 25769803776,
		"DeviceID": 0,
		"DeviceName": "GeForce RTX 3090",
		"HasMonitorConnected": 0,
		"SMX": 82,
		"SM_major": 8,
		"SM_minor": 6,
		"UUID": "",
		"VendorID": 0,
		"VendorName": "",
		"pciBusID": 7,
		"pciDeviceId": 0,
		"pciSubSystemId": 0
	}],
	"DriverVersion": "",
	"ErrorString": "",
	"NvmlInitialized": 1,
	"NvmlLoaded": 1
}
```

**"DriverVersion":""**
with older diriver version:
```
{
	"CudaDevices": [{
		"DeviceGlobalMemory": 8589934592,
		"DeviceID": 0,
		"DeviceName": "GeForce RTX 3070",
		"HasMonitorConnected": 0,
		"SMX": 46,
		"SM_major": 8,
		"SM_minor": 6,
		"UUID": "GPU-f8467b26-8af1-714d-b4e6-c149281c1795",
		"VendorID": 5208,
		"VendorName": "Gigabyte",
		"pciBusID": 5,
		"pciDeviceId": 612634846,
		"pciSubSystemId": 1078727768
	}],
	"DriverVersion": "457.51",
	"ErrorString": "",
	"NvmlInitialized": 0,
	"NvmlLoaded": 1
}
```

**Comment by EvgeniyKorepov** [comment]
Looks like NVIDIA released the driver with bugs. Mass Complaints https://www.nvidia.com/en-us/geforce/forums/game-ready-drivers/13/417924/geforce-46079-game-ready-driver-feedback-thread-re/

**Comment by Squall-Leonhart** [comment]
Already fixed by 3.0.5.5, but explanation is here

https://www.reddit.com/r/NiceHash/comments/k9tfhn/dont_update_to_todays_geforce_46079_driver/gfbd2ja/
